### ISSUE TITLE
Problem with integrated gpu Ryzen 7 4800H?

### ISSUE THREAD

**Issue by HintringerFabian** [brichard19/BitCrack#321]
I think i might have discovered a problem somewhere... or i have the best laptop worldwide
Windows 10 Pro
Ryzen 7 4800H
GTX 1660TI
16 GB RAM

LOOK AT THE FCKING HIGH MKeys/s

.\clBitCrack.exe -d 1 -i .\addresses.txt -b 256 -t 256 -p 1024

[2021-04-08.12:57:31] [Info] Compression: compressed
[2021-04-08.12:57:31] [Info] Starting at: 0000000000000000000000000000000000000000000000000000000000000001
[2021-04-08.12:57:31] [Info] Ending at:   FFFFFFFFFFFFFFFFFFFFFFFFFFFFFFFEBAAEDCE6AF48A03BBFD25E8CD0364140
[2021-04-08.12:57:31] [Info] Counting by: 0000000000000000000000000000000000000000000000000000000000000001
[2021-04-08.12:57:31] [Info] Compiling OpenCL kernels...
[2021-04-08.12:57:39] [Info] Initializing gfx902
[2021-04-08.12:57:44] [Info] Generating 67,108,864 starting points (2560.0MB)
[2021-04-08.12:57:47] [Info] 10.0%
[2021-04-08.12:57:47] [Info] 20.0%
[2021-04-08.12:57:47] [Info] 30.0%
[2021-04-08.12:57:47] [Info] 40.0%
[2021-04-08.12:57:47] [Info] 50.0%
[2021-04-08.12:57:47] [Info] 60.0%
[2021-04-08.12:57:47] [Info] 70.0%
[2021-04-08.12:57:47] [Info] 80.0%
[2021-04-08.12:57:47] [Info] 90.0%
[2021-04-08.12:57:47] [Info] 100.0%
[2021-04-08.12:57:47] [Info] Done
[2021-04-08.12:57:47] [Info] Loading addresses from '.\addresses.txt'
[2021-04-08.12:57:56] [Info] 1,489,525 addresses loaded (28.4MB)
gfx902           6176 / 6240MB | 1489525 targets 482347.27 MKey/s (323,595,385,438,208 total) [00:11:02]

![image](https://user-images.githubusercontent.com/79401071/114017242-1324c980-986c-11eb-8b48-b4fa9c2b509d.png)

![image](https://user-images.githubusercontent.com/79401071/114017594-7f073200-986c-11eb-9775-74ed53d7db70.png)

**Comment by HintringerFabian** [comment]
![image](https://user-images.githubusercontent.com/79401071/114017183-0011f980-986c-11eb-9e1e-2ce105669e04.png)

**Comment by bill32767** [comment]
Generate a wallet randomly with a known HEX PK... Test and see if it could
find it....

**Comment by HintringerFabian** [comment]
Now i tried to find the private keys from the [adressesTest.txt](https://github.com/brichard19/BitCrack/files/6283900/adressesTest.txt) file.

I was able to find the keys with my normal graphics card (gtx 1660ti) (device 0) but not with my device number 1

The programm was working fine with my gtx 1660 ti and generated this document [moneyDocument.txt](https://github.com/brichard19/BitCrack/files/6283953/moneyDocument.txt).


![image](https://user-images.githubusercontent.com/79401071/114144695-e4632d80-9915-11eb-8208-4dd31846fafa.png)

**Comment by websonder** [comment]
its showing too much speed, is it real speed?

**Comment by Uzlopak** [comment]
No. Your gpu ran out of memory. If this happens the OpenCL code is basically empty loops and that's why it is showing such absurd high MKeys. Reduce -p to 256 and try again.

Also the shown used VRAM is not correct. Don't trust that value either.
